barf.go

```diff
@@ -150,6 +159,17 @@ func Beck() error {
 		server.Beckoned = &valid
 		shutdown()
 	}()
+
+	// start server with https enabled
+	if server.Augment.UseHTTPS {
+		logger.Info(fmt.Sprintf("BARF server started at https://%s:%s", server.Augment.Host, server.Augment.Port))
+		err := server.HTTP.ListenAndServeTLS(server.Augment.SSLCertFile, server.Augment.SSLKeyFile)
```

Comment:
I try to avoid using else statements generally, but this is a good use case for one.

I also think the `return nil` should be outside the else block.

Would make necessary changes

Reply:
That's correct....the `nil` should be outside. Thank you for spotting 🤝 and likewise...I always avoid else statements.